packages/driver/cypress/e2e/commands/sessions/sessions.cy.js

```diff
@@ -59,7 +59,7 @@ describe('cy.session', { retries: 0 }, () => {
         .then(async () => {
           cy.spy(Cypress, 'action').log(false)
 
-          await Cypress.action('runner:test:before:run:async', {})
+          await Cypress.action('runner:test:before:run:async', {}, Cypress.state('test'))
```

Comment:
need this change?

Reply:
Some of the test details are now used, thus, these tests needed to pass the test to the action.

Reply:
I'm only seeing changes in this file around `runner:test:before:run:async`, are we passing this in other places already?

Reply:
Yes, Cypress code passes the `runnable`. Updated to `runnable` instead of `test` https://github.com/cypress-io/cypress/pull/25462/commits/c90abe77a969469ca8ed7a039441c10f28c05f35